README.md

```diff
@@ -2,51 +2,28 @@
 <h1 align="center"><img alt="cloudquery logo" width=75% src="https://github.com/cloudquery/cloudquery/raw/main/cli/docs/images/logo.png"/></h1>
 <!-- markdownlint-enable MD033 -->
 
-The open source high performance data integration platform designed for security and infrastructure teams.
+CloudQuery is an [open-source](https://github.com/cloudquery/cloudquery),
+high-performance data integration platform for security and infrastructure teams.
 
-CloudQuery extracts, transforms, and loads your cloud assets. CloudQuery enables you to assess, audit, and monitor the configurations of your cloud assets.
+CloudQuery extracts, transforms, and loads configuration from cloud APIs to
+variety of supported destinations such as databases, data lakes, or streaming platforms
+for further analysis.
```

Comment:
```suggestion
for further analysis.

### Installation

See the **[Quickstart guide](https://www.cloudquery.io/docs/quickstart)** for instructions how to start syncing data with CloudQuery.
```

I'd add a nice bold link to the quickstart guide near the top here, since I think that's what a lot of people will be looking for.